android/app/src/main/java/com/reactnativenavigation/params/parsers/SlidingOverlayParamsParser.java

```diff
@@ -11,6 +11,9 @@ public SlidingOverlayParams parse(Bundle bundle) {
         final SlidingOverlayParams result = new SlidingOverlayParams();
         result.screenInstanceId = bundle.getString("screen");
         result.navigationParams = new NavigationParams(bundle.getBundle("navigationParams"));
+        result.autoDismissTimerSec = bundle.containsKey("autoDismissTimerSec")
+                ? bundle.getInt("autoDismissTimerSec")
+                : null;
```

Comment:
Not anymore because I need some way of disabling the auto dismiss. Also, this matches the iOS implementations. Doing this check in the native code gives us more flexibility in the javascript API.